src/main/java/com/sap/sgs/phosphor/fosstars/tool/github/MarkdownReporter.java

```diff
@@ -1,5 +1,184 @@
 package com.sap.sgs.phosphor.fosstars.tool.github;
 
-public class MarkdownReporter extends AbstractReporter {
+import com.sap.sgs.phosphor.fosstars.model.value.RatingValue;
+import com.sap.sgs.phosphor.fosstars.tool.format.PrettyPrinter;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.text.SimpleDateFormat;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import org.apache.commons.io.IOUtils;
+
+/**
+ * This reporter takes a number of projects and generates a markdown report.
+ */
+public class MarkdownReporter extends AbstractReporter<GitHubProject> {
+
+  /**
+   * A formatter for rating values.
+   */
+  private static final  PrettyPrinter PRETTY_PRINTER = new PrettyPrinter();
+
+  /**
+   * A file where a report is going to be stored.
+   */
+  private static final String REPORT_FILENAME = "README.md";
+
+  /**
+   * A template for a table row in the report.
+   */
+  private static final String PROJECT_LINE_TEMPLATE
+      = "| %URL% | %SCORE% | %LABEL% | %CONFIDENCE% | %DATE% |";
+
+  /**
+   * A date formatter.
+   */
+  private static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("MMM d, YYYY");
```

Comment:
Please consider using default Locale.US here

Reply:
Yup, that's right.